### ISSUE TITLE
Radio buttons in "Cancel Purchase form" are not grouped in radio groups

### ISSUE THREAD

**Issue by ciampo** [Automattic/wp-calypso#48403]
#### Steps to reproduce

1. Create a site with any paid plan
2. Visit `/me/purchases`
3. Select the plan that was purchased at point 1
4. Click on the "Remove WordPress.com [plan]" (last item in the page)
5. A dialog asking for the cancel reason appears, with two questions that can be answered by selecting a radio input

#### What I expected

The radio inputs for each question should be part of the same [radio groups](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#Defining_a_radio_group) (e.g. share the same input `name`). The user should be able to focus on each radio group by pressing the `tab key`, and navigate through each radio option by pressing the keyboard arrows

#### What happened instead

Each radio input is a separate input, and there are no radio groups. The keyboard experience is different from [what a user may expect when interacting with radio groups](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-14).

#### Browser / OS version

N/A

#### Screenshot / Video

![image](https://user-images.githubusercontent.com/1083581/102340087-61008380-3f96-11eb-9b6c-ad93e4706fb4.png)

#### Context / Source

https://github.com/Automattic/wp-calypso/blob/be35e00cdf6c7a6b093047a1434a5ed070a91b3b/client%2Fcomponents%2Fmarketing-survey%2Fcancel-purchase-form%2Findex.jsx#L358-L367

**Comment by vishnugopal** [comment]
This is fixed in #53691
